genesyscloud/resource_genesyscloud_architect_schedulegroups.go

```diff
@@ -144,6 +157,7 @@ func readArchitectScheduleGroups(ctx context.Context, d *schema.ResourceData, me
 		}
 
 		d.Set("name", *scheduleGroup.Name)
+		d.Set("division_id", *scheduleGroup.Division.Id)
```

Comment:
Is this always guaranteed to bet set at this point?

Reply:
I haven't encountered any issue when testing with adding new division and with using home division.